torch/testing/_internal/opinfo/core.py

```diff
@@ -1213,6 +1215,11 @@ def get_decorators(self, test_class, test_name, device, dtype):
         return result
 
     def supported_dtypes(self, device_type):
+        if TEST_WITH_TORCHINDUCTOR:
```

Comment:
This seems like a kludge. Maybe Inductor would be better modeled as two device types, `InductorCPU` and `InductorCUDA`?

Reply:
Dropped this in the new version. Skipping `complex` from CI cmd instead.